packages/fxa-auth-server/lib/server.js

```diff
@@ -217,6 +217,11 @@ async function create(log, error, config, routes, db, statsd, glean) {
         clientAddressIndex = 0;
       }
 
+      log.info('server.clientAddress', {
+        remote_address_chain: remoteAddressChain.join(','),
```

Comment:
Is this going to log full IPs (do we want this in stage only)?

Reply:
No concern from me, fwiw